pokete_classes/fightmap.py

```diff
@@ -174,31 +175,27 @@ def get_attack(self, attack_obs):
                 self.atk_info_box.add(self, 27, self.height - 7)
             self.show()
             while True:#158
-                if _ev.get() in ["'s'", "'w'"]:
-                    self.box.input(_ev.get())
+                action = get_action()
+                if action in [Action.UP, Action.DOWN]:
+                    self.box.input(action)
                     self.rechar_atk_info_box(attack_obs)
                     self.show()
-                    _ev.clear()
-                elif _ev.get() in [f"'{i + 1}'" for i in
-                                   range(len(attack_obs))] + ["Key.enter"]:
-                    attack = attack_obs[self.box.index.index
-                                        if _ev.get() == "Key.enter"
-                                        else int(_ev.get().strip("'")) - 1]
-                    _ev.clear()
+                elif not action == None and action.value in [i + Action.ACT_1.value for i in range(len(attack_obs))] and action.value < Action.ACT_9.value or action == Action.ACCEPT:
+                    attack = attack_obs[
+                        self.box.index.index if action == Action.ACCEPT else action.value - Action.ACT_1.value
+                    ]
                     if attack.ap == 0:
                         continue
                     break
-                elif _ev.get() == "'i'":
-                    _ev.clear()
+                elif action == Action.HELP:
```

Comment:
```suggestion
                elif action == Action.INFO:
```

Reply:
What is Action.INFO? I hate this github code review view